keps/sig-api-machinery/365-paginated-lists/README.MD

```diff
@@ -40,7 +122,18 @@ Relevant issues:
 
 * https://github.com/kubernetes/kubernetes/issues/2349
 
-## Terminology
+### Goals
+
+- Expose API chunking mechanism as opt-in for clients
+
+### Non-Goals
```

Comment:
is a paginated watch cache a goal or non-goal?

Reply:
it's a goal - added

Reply:
Watch cache is an optimization only; the goal would be something like consistent semantics no matter how the request is serviced, or user-selectable semantics where some choices may be more efficient.

(also, missing push?)